uaclient/entitlements/repo.py

```diff
@@ -173,10 +174,14 @@ def setup_apt_config(self) -> None:
                 'No specific resourceToken present. Using machine token'
                 ' as %s credentials', self.title)
             token = self.cfg.machine_token['machineToken']
-        if directives.get('aptKey'):
-            logging.debug(
-                "Ignoring aptKey directive '%s'", directives.get('aptKey'))
-        keyring_file = os.path.join(apt.KEYRINGS_DIR, self.repo_key_file)
+        aptKey = directives.get('aptKey')
+        if not aptKey:
+            logging.warning(
```

Comment:
If we aren't going to install a key, would it be better to just error out here (rather than continue on to enable a repo which won't be useable)?

Reply:
raising a UserFacingError here

Reply:
+1